test/parallel_api/experimental/radix_sort_esimd.pass.cpp

```diff
@@ -119,7 +162,7 @@ void test_all_view(std::size_t size)
     EXPECT_EQ_RANGES(ref, input, msg.c_str());
 }
 
-template<typename T>
```

Comment:
I think it is enough to test different USM allocation only inside `test_usm`.

Reply:
This test organized like all out other tests from this point of view.
I don't think that we should have some special in this test.

Reply:
Compilation time of that test is already huge and we agreed not to make "combinatoric explosion" of the cases. I believe this case does not bring much value and it is better not to include it.